### PR TITLE
Jr photogram industrial 2

### PR DESCRIPTION
**Pull request by jb520**
Generated comments, follow_requests, and likes tables. Created sample_data task.

### CODE REVIEW

**Review by MayaS1111 — APPROVED** [review]
Looks great!

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
+
+  validates :fan_id, uniqueness: { scope: :photo_id, message: "has already liked this photo" }
```
Comment (thierrychau) [inline app/models/like.rb:25]: Great job!

**Review by thierrychau — APPROVED** [review]
Great job, good to merge!

**Comment by FadiBaker92** [comment]
Looks good! Great job Jennifer.

**Pull request merged**